cronjob/templates/secrets.yaml

```diff
@@ -0,0 +1,14 @@
+## The secret resource is required in order to hide secrets within the K8s UI. If we skip the usage of secrets yaml,
+## sensitive information would be shown in plain text i.e. in the Google console.
+apiVersion: v1
+kind: Secret
+metadata:
+  # The identifier of this Secret, unique within a namespace.
+  # https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+  name: {{ .Chart.Name }}-secret
```

Comment:
name should be something like:

```
name: {{ template "cronjob.fullname" . }}
```

(secretKeyRef.name in cronjob.yaml needs to be adjusted then as well)

Reply:
https://github.com/commercetools/k8s-charts/pull/1/commits/3ee7bd2cba21e037cef66c8efb078cf686a27d0e